projects/addon-charts/components/line-chart/line-chart.component.ts

```diff
@@ -40,6 +40,7 @@ import {TuiLineChartHintDirective} from './line-chart-hint.directive';
     viewProviders: [tuiHintOptionsProvider({direction: 'top', hideDelay: 0})],
 })
 export class TuiLineChartComponent {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
     private readonly _hovered$ = new Subject<number>();
```

Comment:
It's a private property, you can just rename it to something like `hover$`

Reply:
But this class already has `get hover$`

Reply:
fixed